Sming/Libraries/ModbusMaster/samples/generic/component.mk

```diff
@@ -1,7 +1,2 @@
 ARDUINO_LIBRARIES := ModbusMaster
 DISABLE_SPIFFS = 1
-
-# Configurable Modbus response timeout
-COMPONENT_VARS += MB_RESPONSE_TIMEOUT
```

Comment:
Where did that go?

Reply:
It's defined in the ModbusMaster library's `component.mk` file.